.github/workflows/podvm_builder.yaml

```diff
@@ -61,6 +61,6 @@ jobs:
           "PROTOC_VERSION=3.11.4"
           "RUST_VERSION=1.66.0"
           "CAA_SRC=https://github.com/confidential-containers/cloud-api-adaptor"
-          "CAA_SRC_BRANCH=staging"
+          "CAA_SRC_REF=staging"
```

Comment:
Looks like we are using a mix of CAA_SRC_BRANCH and CAA_SRC_REF which is a mess :-( 

We should make it consistent and uniform across the different Dockerfiles, workflows and doc as part of a separate PR.

@stevenhorsman any preference on whether to use CAA_SRC_BRANCH  or CAA_SRC_REF ?

Reply:
I think that `CAA_SRC_BRANCH` was the original solution and then @wainersm introduced `CAA_SRC_REF` (maybe to other docker files?) for helping to run them against PRs. It's worth getting Wainer's confirmation, but I guess that using (and documenting ) `REF` consistently 'covers' more cases? Not a strong feeling though.